Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -158,16 +157,20 @@ public function storeOutput()
             $this->logger->debug("Default bucket " . $uploadTablesOptions["bucket"]);
         }
 
-        $writer->uploadTables($this->dataDirectory . "/out/tables", $uploadTablesOptions, $systemMetadata);
         try {
-            $writer->uploadFiles($this->dataDirectory . "/out/files", ["mapping" => $outputFilesConfig]);
-        } catch (ManifestMismatchException $e) {
-            $this->logger->warning($e->getMessage());
-        }
+            $writer->uploadTables($this->dataDirectory . "/out/tables", $uploadTablesOptions, $systemMetadata);
+            try {
```

Comment:
nemohly, protoze u uploadFiles se handluj manifest mismatch jako warning, kdezto u uploadtables se handluje jako error. Otazka je proc a to bohuzel nevim, nicmene bylo to tak i predtim a nechtel jsem to menit.